pcsx2/IopBios.cpp

```diff
@@ -989,22 +1061,37 @@ namespace R3000A
 			EXPORT_H( 14, Kprintf)
 		END_MODULE
 
-		MODULE_2(ioman, iomanx)
-			EXPORT_H(  4, open)
-			EXPORT_H(  5, close)
-			EXPORT_H(  6, read)
-			EXPORT_H(  7, write)
-			EXPORT_H(  8, lseek)
-			EXPORT_H( 10, remove)
-			EXPORT_H( 11, mkdir)
-			EXPORT_H( 12, rmdir)
-			EXPORT_H( 13, dopen)
-			EXPORT_H( 14, dclose)
-			EXPORT_H( 15, dread)
-			EXPORT_H( 16, getStat)
-		END_MODULE
+		// Special case with ioman and iomanX
+		// They are mostly compatible excluding stat structures
+		if(libname == "ioman" || libname == "iomanx")
+		{
+			const bool use_ioman = libname == "ioman";
+			using namespace ioman;
+				switch(index)
```

Comment:
weird indenting

Reply:
I guess it's not wise to rely on clang-format when the code is in a clang-format off region lol.